File: atcoder.jp/contests/abc213/tasks/abc213_b/main.ts
import { readFileSync } from "fs";
let inputLinesIndex = 0;
const inputLines = readFileSync("/dev/stdin", "utf8").split("\n");
const input = () => inputLines[inputLinesIndex++];

const enumerate = <T>(array: T[], start: number = 0): [number, T][] =>
  array.map((v, i) => [start + i, v]);
const fst = <T>(x: [T, any]) => x[0];

const main = () => {
  const N = +input();
  const A = input().split(" ").map(Number);

  const ans = fst(enumerate(A, 1).sort(([, a], [, b]) => a - b)[N - 2]);
  console.log(ans);
};

main();

File: atcoder.jp/contests/abc213/tasks/abc213_b/main.py
from typing import Any, Tuple, TypeVar

_T = TypeVar("_T")


def fst(x: Tuple[_T, Any]) -> _T:
    return x[0]


def snd(x: Tuple[Any, _T]) -> _T:
    return x[1]


N = int(input())
A = list(map(int, input().split()))

ans = fst(sorted(enumerate(A, start=1), key=snd)[-2])
print(ans)
